Doc/whatsnew/3.2.rst

```diff
@@ -1060,15 +1060,15 @@ of nearly equal quantities:
 
 The :func:`~math.erf` function computes a probability integral or `Gaussian
 error function <https://en.wikipedia.org/wiki/Error_function>`_.  The
-complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``:
+complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``::
```

Comment:
@berkerpeksag I did perceive Marco as lecturing, but rather giving the context for why he had a doubt about your suggestion.

Marco: the difference here with what PEP8 is talking about is that we are now (starting from approximately last year's PyCon sprint) actually trying to execute doctests, where we've never been worried about that in the past (and the use of : and :: may have been somewhat random).  So as far as skipping doctests or not, there is no true pre-existing style to follow.  We ought instead to figure out what we want to do documentation wide, and do it.  And I do like Berker's suggestion, since it makes it explicit.  I myself can never remember which colon form does which.